pandas/tests/internals/test_managers.py

```diff
@@ -68,3 +84,22 @@ def test_series_creation():
     result = s_array._as_manager("block")
     assert isinstance(result._mgr, SingleBlockManager)
     tm.assert_series_equal(result, s_array)
+
+
+@pytest.mark.parametrize("manager", ["block", "array"])
+def test_array_manager_depr_env_var(manager):
+    # GH#55043
+    test_env = os.environ.copy()
+    test_env["PANDAS_DATA_MANAGER"] = manager
+    response = subprocess.run(
+        [sys.executable, "-c", "import pandas"],
+        capture_output=True,
+        env=test_env,
+        check=True,
+    )
+    msg = "<string>:1: FutureWarning: Using ArrayManger through the environment"
+    stderr_msg = response.stderr.decode("utf-8")
+    if manager == "block":
+        assert msg not in stderr_msg, stderr_msg
```

Comment:
do we want to tell them that the environment variable is deprecated even if they are using "block"?

Reply:
I could go either way here - I anticipate it would be quite rare people are using the env variable, especially to specify block. Any preference?

Reply:
very slight preference for warning but NBD